### PR TITLE
Sensu EE licensing

### PR DESCRIPTION
**Pull request by palourde**
## Description
It adds the documentation around the licensing for Sensu EE. For reference, I added this documentation to Sensu Core 2.0, which will live in a branch named `sensu-enterprise-edition` until the Success team complete their refactoring with the new product names.

Also, we make sure to display the Enterprise section, in the left-side menu, at the end.

## Motivation and Context
Closes https://github.com/sensu/sensu-enterprise-go/issues/16

## How Has This Been Tested?
Locally tested!

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] New doc/guide
- [ ] Fixing errata
- [ ] Update (Add missing or refresh existing content)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] All tests have passed.

### CODE REVIEW

File: content/sensu-core/2.0/enterprise/licensing.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+sensuctl license info
+```
+[1]: ../../reference/sensuctl
+
+## Expiration of your license
+
+Sensu Backend will start issuing regular warnings via the log messages 30 days
+before the expiration of your license.
+
+If your license expire and you do not renew it, the Sensu EE
```
Comment (nikkictl) [inline content/sensu-core/2.0/enterprise/licensing.md]: If your license _expires_...

File: content/sensu-core/2.0/enterprise/licensing.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+A Sensu Enterprise license allows customers to enable additional features in
+Sensu Enterprise Edition (EE) compared to Sensu Core Edition (CE). Licenses are
+cryptographically signed documents that can easily be managed with
+[sensuctl][1].
+
+## Installing and renewing your license
+
+The license file will be provided as a JSON file, which should be saved on your
+local computer. We will then use `sensuctl` to upload it to Sensu Backend:
```
Comment (nikkictl) [inline content/sensu-core/2.0/enterprise/licensing.md]: upload it to _the_ Sensu Backend

File: content/sensu-core/2.0/enterprise/licensing.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+cat <<EOF | sensuctl license install
+{"license":"[...]", "signature":"[...]"}
+EOF
+```
+
+## Viewing your license details
+
+The details of the installed Sensu license can easily be displayed using
+`sensuctl`, which will provide you with information like your current plan, the
+features enabled and your license expiry date.
```
Comment (nikkictl) [inline content/sensu-core/2.0/enterprise/licensing.md]: _enabled features_

**Review by nikkictl — COMMENTED** [review]
Overall, great outline! Just a few points that might make readability a little easier.

**Review by echlebek — APPROVED** [review]

File: content/sensu-core/2.0/enterprise/licensing.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+---
+title: "Licensing"
+description: "Managing Sensu EE license."
```
Comment (grepory) [inline content/sensu-core/2.0/enterprise/licensing.md]: Do we need an article there? "Managing a Sensu EE license"

File: content/sensu-core/2.0/enterprise/licensing.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+platformContent: false
+menu:
+  sensu-core-2.0:
+    parent: enterprise
+---
+
+## How Sensu Enterprise licensing works
+
+A Sensu Enterprise license allows customers to enable additional features in
+Sensu Enterprise Edition (EE) compared to Sensu Core Edition (CE). Licenses are
```
Comment (grepory) [inline content/sensu-core/2.0/enterprise/licensing.md]: ~compared to~ that are not available in

**Review by nikkictl — APPROVED** [review]
giphy

**Review by grepory — COMMENTED** [review]
Couple of minor things, but otherwise, seems fine.

**Pull request merged**